Misc/NEWS.d/next/IDLE/2024-10-12-08-08-37.gh-issue-125349.u0nToN.rst

```diff
@@ -0,0 +1 @@
+Fixed unexpected behavior caused by the input method.
```

Comment:
It seems to be chinese, You should explain in detail in NEWS that the chinese input method is not correct.
Otherwise this will confuse the user because it doesn't know what language the input method is.

Reply:
At the moment, I'm not sure if this problem only occurs with Chinese input methods, and maybe with input methods in other languages. Because the root cause of the problem is not linguistic.